### PR TITLE
fix: resolve Biome linter issues

### LINKED ISSUE #122: misc: fix linter issues

### PR DESCRIPTION
**Pull request by marcinlesek**
## Summary

- Renames `getConfigFileURL` to `getConfigFileUrl` for consistent camelCase naming convention
- Replaces `console.log` calls with `console.info` in `cli.ts` and `init.ts`
- Converts namespace imports (`* as fs`, `* as path`, `* as ts`) to named imports in `getSpecFilePaths.ts` and `getTestContextPaths.ts`
- Fixes unused parameter warnings by prefixing with `_` in Proxy `get` handlers; adds `biome-ignore` comment for intentional empty Proxy target
- Replaces empty `interface PlaywrightPluginOptions {}` with `type PlaywrightPluginOptions = Record<string, never>`
- Simplifies null check `!image || !image.imageId` to `!image?.imageId` using optional chaining
- Removes `as any` cast in `bin.ts` by providing a properly typed Lambda payload with `runId`, `testId`, `bucketName`, and `context`
- Refactors `parsePlaywrightReport` to extract a `buildInvokeResponse` helper, use `??` over `||`, and simplify iteration via nullish coalescing on optional arrays
- Moves `public readonly problem` out of `CheckError` constructor parameter into the class body to satisfy `noParameterProperties` rule
- Updates barrel file `biome-ignore` comment with a clearer explanation

### CODE REVIEW

File: packages/hypertest-plugin-playwright/src/types.ts [hunk 1]
```diff
@@ -2,4 +2,4 @@ export interface PlaywrightCloudFunctionContext {
   grep: string;
 }
 
-export interface PlaywrightPluginOptions {}
+export type PlaywrightPluginOptions = Record<string, never>;
```
Comment (blazbla274) [inline packages/hypertest-plugin-playwright/src/types.ts]: Im not sure but it looks like an old artefact 🤔 PlaywrightPluginOptions is used only in PlaywrightRunnerPlugin. Where option is a prop of option, but options.options is never used.
But I can see that emty options is important for keeping plugins consistent, but still I do not like that we have option inside option property. This structure differ a little from what we have for cloud provider.
I think we should make Plugins interfaces more consistent and define  what is the differance between options settings and config. Lets discuss it in discord :D

File: packages/hypertest-runner-aws-playwright/src/utils/parsePlaywrightReport.ts [hunk 2]
```diff
@@ -31,6 +31,32 @@ interface PlaywrightReport {
...
+  const name = [...currentPath, spec.title].join(' > ');
+  const filePath = spec.file;
+
+  if (result?.status === 'failed') {
+    return {
+      success: false,
+      name,
+      filePath,
+      message: result.error?.message ?? 'Unable to retrieve message',
+      stackTrace: result.error?.stack ?? 'Unable to retrieve stack trace',
```
Comment (blazbla274) [inline packages/hypertest-runner-aws-playwright/src/utils/parsePlaywrightReport.ts:49]: Are we sure that we do not want to have duration for failed tests? Or we can be sure that it will be always undefined or null?

**Review by blazbla274 — DISMISSED** [review]
Approved, but check the coments!

**Comment by marcinlesek** [comment]
Okey, I agreed with both and fixed them @blazbla274:

  1. `PlaywrightPluginOptions` / `options` nesting — removed `PlaywrightPluginOptions` entirely (type, import, and the options.options double-nesting inside `PlaywrightRunnerPlugin`). The public `plugin()` still accepts and validates user options via `OptionsSchema`, it just no longer threads them into the internal plugin instance since there was nothing consuming them. Broader alignment of plugin vs cloud-provider option shapes is worth a separate discussion — let's take it to Discord.
  2. Duration for failed tests — good catch, Playwright does record duration for failing tests. Added `duration` to the `success: false` variant in `TestInvokeResponseSchema` and now pass `result.duration` through in `buildInvokeResponse`. Marked `optional` on the schema side to cover edge cases where a test fails during setup before the body runs.

**Review by blazbla274 — APPROVED** [review]
